src/stream.rs

```diff
@@ -55,14 +55,18 @@ pub struct OutputStreamBuilder {
     config: OutputStreamConfig,
 }
 
-impl Default for OutputStreamConfig {
```

Comment:
I do not know why the `Default` impl is gone. You have refactored that code so you know what you must know what your doing.